docs/generate_db_schema

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+Generate an image of the database schema
+"""
+from sqlalchemy_schemadisplay import create_schema_graph
+
+from anitya.db import meta
+
+
+def write_graph(filename):
```

Comment:
It'd be good to add a docblock here, including the type of the ```filename``` parameter.

Reply:
I copied this from Bodhi, changed only what is imported. :-)